### PR TITLE
fix(web): Directory Structure 

### PR DESCRIPTION
**Pull request by McPizza0**
fixes the directory structure for the new nextjs app
adds initial shadcn features

### CODE REVIEW

**Comment by McPizza0** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/un/inbox/448?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#448** <a href="https://app.graphite.dev/github/pr/un/inbox/448?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#447** <a href="https://app.graphite.dev/github/pr/un/inbox/447?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#446** <a href="https://app.graphite.dev/github/pr/un/inbox/446?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#445** <a href="https://app.graphite.dev/github/pr/un/inbox/445?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#444** <a href="https://app.graphite.dev/github/pr/un/inbox/444?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/styles/globals.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+@layer tw_components_utilities {
+  @tailwind components;
+  @tailwind utilities;
+}
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (BlankParticle) [inline apps/web/src/styles/globals.css:15]:
```suggestion

```
these are duplicates

File: apps/web/src/styles/globals.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+@layer base {
```
Comment (BlankParticle) [inline apps/web/src/styles/globals.css:17]:
```suggestion
@layer tw_base {
```

File: apps/web/src/styles/globals.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    --destructive: 0 62.8% 30.6%;
+    --destructive-foreground: 210 40% 98%;
+
+    --border: 217.2 32.6% 17.5%;
+    --input: 217.2 32.6% 17.5%;
+    --ring: 212.7 26.8% 83.9%;
+  }
+}
+
+@layer base {
```
Comment (BlankParticle) [inline apps/web/src/styles/globals.css:81]:
```suggestion
@layer tw_base {
```

**Review by BlankParticle — CHANGES_REQUESTED** [review]
Also you can just edit `tsconfig.json` and change the alias `@/` to `./src/` so you wont need to do `@/src/` everytime

**Comment by McPizza0** [comment]
@/src is still good to have
as the app gets more complex, we may need additional root folders
we reduce confusion by always having the full path relative to app

**Comment by BlankParticle** [comment]
> @/src is still good to have as the app gets more complex, we may need additional root folders we reduce confusion by always having the full path relative to app

ok if you say so

**Review by BlankParticle — APPROVED** [review]

**Comment by BlankParticle** [comment]
### Merge activity

* **May 14, 12:35 PM EDT**: @BlankParticle merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/448).

**Pull request merged**